src/app/window.cpp

```diff
@@ -74,6 +75,7 @@ QTabWidget *MainWindow::tabs_widget()
     launcher->setObjectName("Launcher");
     SettingsTab *settings = new SettingsTab(this);
     settings->setProperty("prevent_disable", true);
+    CameraTab *camera = new CameraTab(this);
```

Comment:
can you do `camera->setObjectName("Camera");` after this

the object name is used for labeling the different pages to enable/disable

Reply:
Ok, I fixed this. Not sure if it's just a delay that it's not showing up.